### ISSUE TITLE
convert schema facet to tables and fix truncation

### ISSUE THREAD

**Issue by curlyman72** [dashbase/dashbase-engine-web#196]
The schema facet is a table and should be built with a table for accessibility.  since the facet lives in the side bar, there's not much we can do for truncation due to the limited real estate.  To fix the header truncation, will look into converting the labels into icons [A] [#] [</>] should be intuitive enough with the addition of alt/title tags.

firefox and chrome on windows/linux does not like the inline divs and pushes the right 3 divs below the far left one.
